app/stores/SubscriptionsStore.ts

```diff
@@ -0,0 +1,18 @@
+import Subscription from "~/models/Subscription";
+import BaseStore, { RPCAction } from "./BaseStore";
+import RootStore from "./RootStore";
+
+export default class SubscriptionsStore extends BaseStore<Subscription> {
+  actions = [
+    RPCAction.Info,
+    // list is disabled.
+    // RPCAction.List
+    RPCAction.Create,
```

Comment:
We could remove `subscriptions.list` route entirely before finalizing. Right now I find it really useful for testing.